code/datums/martial/reverbpalm.dm

```diff
@@ -66,6 +66,10 @@
 
 /datum/martial_art/reverberating_palm/can_use(mob/living/carbon/human/H)
 	var/obj/item/bodypart/r_arm/robot/seismic/R = H.get_bodypart(BODY_ZONE_R_ARM)
+	var/turf/current = get_turf(H)
+	for(var/obj/outside in current.contents)
```

Comment:
Checking `!isturf(H.loc)` would be much faster

Reply:
oops didnt know loc was chill like that thanks